go/consensus/tendermint/roothash/roothash.go

```diff
@@ -419,8 +426,15 @@ func (sc *serviceClient) DeliverEvent(ctx context.Context, height int64, tx tmty
 			continue
 		}
 
-		if err = sc.processFinalizedEvent(ctx, height, ev.RuntimeID, &ev.FinalizedEvent.Round, true); err != nil {
-			return fmt.Errorf("roothash: failed to process finalized event: %w", err)
+		if tr := sc.trackedRuntime[ev.RuntimeID]; tr != nil {
```

Comment:
Same as above, consider using a continue statement and not emit a debug entry each time.